test/boulder-config-next.json

```diff
@@ -39,9 +39,13 @@
     "rsaProfile": "rsaEE",
     "ecdsaProfile": "ecdsaEE",
     "debugAddr": "localhost:8001",
-    "Key": {
-      "File": "test/test-ca.key"
-    },
+    "Issuers": [ {
+      "File": "test/test-ca.key",
+      "CertFile": "test/test-ca2.pem"
```

Comment:
Shouldn't test-ca1 be the default issuer so you don't have to change all the other settings? And you can swap it back to ca2 first when you need to test.